pkg/kn/traffic/compute.go

```diff
@@ -44,6 +44,9 @@ func newServiceTraffic(traffic []servingv1.TrafficTarget) ServiceTraffic {
 
 func splitByEqualSign(pair string) (string, string, error) {
 	parts := strings.Split(pair, "=")
+	if len(parts) == 1 {
+		return latestRevisionRef, parts[0], nil
```

Comment:
Can we get this part covered by tests please? It can be only specific test for the `splitByEqualSign` function.

Reply:
Will add